### ISSUE TITLE
Incorrect propType warning on `labelStyle`

### ISSUE THREAD

**Issue by hdfelix** [kyoyadmoon/react-native-icon-checkbox#1]
@kyoyadmoon I am getting the following warning:
`Warning: Failed propType: Invalid prop 'labelStyle' of type 'number' supplied to 'CheckBox', expected 'object'.`

Is the propType warning incorrect or am I missing something?

Here's my checkbox usage:

```
View style={styles.container}>
 <CheckBox
   checked={this.state.isChecked}
   color={DARK_GREY}
   label={this.props.subTitle}
   labelStyle={styles.labelStyle}
   onPress={this._handlePressCheckBox}
   size={28}
   style={styles.checkBox}
 />
</View>
```

Here's styles.labelStyle:

```
const styles = StyleSheet.create({
  container: {
    flex: 1,
    flexDirection: 'row',
    alignItems: 'center',
    backgroundColor: WHITE,
    justifyContent: 'flex-start',
    paddingLeft: 20
  },
  labelStyle: {
    color: DARK_GREY,
    fontSize: 20,
    fontStyle: 'italic',
  },
});
```

The style is being applied correctly; the issue is just the warning:

![screen shot 2016-04-27 at 9 45 40 am](https://cloud.githubusercontent.com/assets/5149861/14860258/da9856ae-0c5c-11e6-829f-157041825dbe.png)

**Comment by kyoyadmoon** [comment]
I upgraded react-native version to 0.32.0 with React version 15.3.1 and valiate style props with Text.propTypes.style.
You can try to update to v1.2.0. It should be fixed.
